helix-term/src/commands/typed.rs

```diff
@@ -2454,6 +2455,37 @@ fn yank_diagnostic(
     Ok(())
 }
 
+fn read_file_info_buffer(
```

Comment:
There's a typo here info->into. But I think we can just call this `read`, it should be a little easier to find given the command name is `:read`